app/services/foreman/template_snapshot_service.rb

```diff
@@ -196,6 +200,17 @@ def ubuntu_autoinst4dhcp
     def rhel9_dhcp
       host = FactoryBot.build(:host_for_snapshots_ipv4_dhcp_rhel9,
         name: 'snapshot-ipv4-dhcp-rhel9',
+        'ntp-server': '127.0.0.1',
+        subnet: FactoryBot.build(:subnet_ipv4_dhcp_for_snapshots),
+        interfaces: [ipv4_interface])
+      host.define_singleton_method(:managed_interfaces) { interfaces }
+      define_host_params(host)
+    end
+
+    def rhel9_dhcp_ntp_multiple
+      host = FactoryBot.build(:host_for_snapshots_ipv4_dhcp_ntp_multiple_rhel9,
+        name: 'snapshot-ipv4-dhcp-ntp-multiple-rhel9',
+        'ntp-server': ['127.0.0.1', '192.168.0.0'],
```

Comment:
`ntp-server` is a host parameter, not a property of the host object. You need to add it with the rest of the parameters in `define_host_params` method: https://github.com/theforeman/foreman/blob/b1343ecd50296d0dddcdee076a4e078236c43f2f/app/services/foreman/template_snapshot_service.rb#L97